src/pzt/default/sections/20-operationalize/10-renewal/20-deployments/kubernetes/20-issuer/jwk.mdx

```diff
@@ -44,9 +44,9 @@ Create the secret in your cluster, then apply your `StepIssuer` resource.
 
 <CodeBlock
   language="shell"
-  copyText=" kubectl create secret generic my-ca-provisioner-password --from-literal=password=Y4nys7f11 && kubectl apply -f my-ca-issuer.yaml"
+  copyText="kubectl create secret generic my-ca-provisioner-password --from-literal=password=Y4nys7f11 && kubectl apply -f my-ca-issuer.yaml"
```

Comment:
This is the second time I think someone has deleted this space in a PR. :laughing: It's intentional, since we have the password in plain text in the command. The space keeps it out of the command history in a lot of shells.